bin/ruby-build

```diff
@@ -1190,6 +1193,7 @@ fi
 
 if [ -z "$RUBY_BUILD_MIRROR_URL" ]; then
   RUBY_BUILD_MIRROR_URL="https://dqw8nmjcqpjn7.cloudfront.net"
+  RUBY_BUILD_DEFAULT_MIRROR=1
```

Comment:
This is OK, but the `else` branch of this conditional has to also set `RUBY_BUILD_DEFAULT_MIRROR`, but to an empty string. This is so that RUBY_BUILD_DEFAULT_MIRROR can't accidentally be inherited from the environment.